src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -179,17 +179,19 @@ class SubjectHeading extends React.Component {
 
     return (
       <li data={`${subjectHeading.uuid}, ${container}`} className={`subjectHeadingRow ${ open || children ? "openSubjectHeading" : ""}`}>
-        <a  className={`subjectHeadingRow ${ open || children ? "openSubjectHeading" : ""} + ${this.props.nested ? ' nestedSubjectHeading' : ''}`} >
-          <span style={{'paddingLeft': `${20*indentation}px`}} className="subjectHeadingLabelAndToggle">
-            <span onClick={container !== 'context' ? this.toggleOpen : () => {} } className="subjectHeadingToggle" >{desc_count > 0 ? (!open ? '+' : '-') : null}</span>
-            <span className="subjectHeadingLabel" onClick={this.linkToShow}><span>{rest}</span>{rest === '' ? '' : ' -- ' }<span className='emph'>{emph}</span></span>
-          </span>
-          <span className="subjectHeadingAttribute titles">{`${bib_count}`}</span>
-          <span className="subjectHeadingAttribute narrower">{`${desc_count}`}</span>
-          { open && narrower.length > 1 && uuid.length > 0 && (container !== 'context' || location.pathname.includes(uuid))
-            ? <SortButton sortBy={sortBy} handler={this.sortHandler} />
+        <a>
+          <div  className={`subjectHeadingInfo subjectHeadingRow ${ open || children ? "openSubjectHeading" : ""} + ${this.props.nested ? ' nestedSubjectHeading' : ''}`} >
+            <span  className="subjectHeadingLabelAndToggle">
+              <span style={{'paddingLeft': `${20*indentation}px`}} onClick={container !== 'context' ? this.toggleOpen : () => {} } className="subjectHeadingToggle" >{desc_count > 0 ? (!open ? '+' : '-') : null}</span>
```

Comment:
According to Edwin, it is less accessible for there to be an `onClick` on a span than on an `a` tag or button. Here is a code pen I found quickly that shows how to override button styling https://codepen.io/andybelldesign/pen/Vxpjvo Basically `border: none; text-decoration: none;` will do that. I'm in favor of that.